File: Backend/Infrastructure.MyShop/Models/DTO/AccountDTO/UserEditDTO.cs
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Infrastructure.MyShop.Models.DTO.AccountDTO
{
    public class UserEditDTO
    {
        public long id { get; set; }
        public string Email { get; set; }
        public string Firstname { get; set; }
        public string Lastname { get; set; }
        public IFormFile Image { get; set; }
        public string phoneNumber { get; set; }
        public string Password { get; set; }
        public string ConfirmPassword { get; set; }
        public string? Role { get; set; }
    }
}

File: Backend-Java/src/main/java/org/example/Infrastructure/dto/accountDTO/UserEditDTO.java
package org.example.Infrastructure.dto.accountDTO;

import jakarta.annotation.Nullable;
import jakarta.validation.constraints.Null;
import lombok.Data;
import org.springframework.web.multipart.MultipartFile;

@Data
public class UserEditDTO {
    private int id;
    private String email;
    private String firstname;
    private String lastname;
    private MultipartFile imageUpload;
    private String phoneNumber;
    private String role;
}
